### ISSUE TITLE
`NullPointerException: Cannot invoke "com.github.einjerjar.mc.keymap.keys.layout.KeyLayout.keys()" because "layout" is null`

### ISSUE THREAD

**Issue by SettingDust** [Einjerjar/keymap#31]
```
[00:32:33] [Render thread/ERROR]: Reported exception thrown!
net.minecraft.class_148: mouseClicked event handler
	at net.minecraft.class_437.method_25412(class_437.java:497) ~[client-intermediary.jar:?]
	at net.minecraft.class_312.method_1601(class_312.java:94) ~[client-intermediary.jar:?]
	at net.minecraft.class_312.method_22686(class_312.java:165) ~[client-intermediary.jar:?]
	at net.minecraft.class_1255.execute(class_1255.java:103) ~[client-intermediary.jar:?]
	at net.minecraft.class_312.method_22684(class_312.java:165) ~[client-intermediary.jar:?]
	at org.lwjgl.glfw.GLFWMouseButtonCallbackI.callback(GLFWMouseButtonCallbackI.java:36) ~[lwjgl-glfw-3.2.2.jar:build 10]
	at org.lwjgl.system.JNI.invokeV(Native Method) ~[lwjgl-3.2.2.jar:build 10]
	at org.lwjgl.glfw.GLFW.glfwPollEvents(GLFW.java:3101) ~[lwjgl-glfw-3.2.2.jar:build 10]
	at com.mojang.blaze3d.systems.RenderSystem.flipFrame(RenderSystem.java:197) ~[client-intermediary.jar:?]
	at net.minecraft.class_1041.method_15998(class_1041.java:310) ~[client-intermediary.jar:?]
	at net.minecraft.class_310.method_1523(class_310.java:1147) ~[client-intermediary.jar:?]
	at net.minecraft.class_310.method_1514(class_310.java:737) [client-intermediary.jar:?]
	at net.minecraft.client.main.Main.main(Main.java:236) [client-intermediary.jar:?]
	at jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:578) ~[?:?]
	at org.quiltmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:475) [quilt-loader-0.18.1-beta.25.jar:?]
	at org.quiltmc.loader.impl.launch.knot.Knot.launch(Knot.java:77) [quilt-loader-0.18.1-beta.25.jar:?]
	at org.quiltmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:24) [quilt-loader-0.18.1-beta.25.jar:?]
Caused by: java.lang.NullPointerException: Cannot invoke "com.github.einjerjar.mc.keymap.keys.layout.KeyLayout.keys()" because "layout" is null
	at com.github.einjerjar.mc.keymap.utils.VKUtil.genLayout(VKUtil.java:41) ~[keymap-0.7.0-beta.01.18.2-fabric.jar.i0:0/:?]
	at com.github.einjerjar.mc.keymap.utils.VKUtil.genLayout(VKUtil.java:26) ~[keymap-0.7.0-beta.01.18.2-fabric.jar.i0:0/:?]
	at com.github.einjerjar.mc.keymap.client.gui.screen.LayoutSelectionScreen.initVks(LayoutSelectionScreen.java:123) ~[keymap-0.7.0-beta.01.18.2-fabric.jar.i0:0/:?]
	at com.github.einjerjar.mc.keymap.client.gui.screen.LayoutSelectionScreen.onInit(LayoutSelectionScreen.java:42) ~[keymap-0.7.0-beta.01.18.2-fabric.jar.i0:0/:?]
	at com.github.einjerjar.mc.widgets.EScreen.method_25426(EScreen.java:46) ~[keymap-0.7.0-beta.01.18.2-fabric.jar.i0:0/:?]
	at net.minecraft.class_437.method_25423(class_437.java:389) ~[client-intermediary.jar:?]
	at net.minecraft.class_310.method_1507(class_310.java:993) ~[client-intermediary.jar:?]
	at net.minecraft.class_458.clickHandler(com/github/einjerjar/mc/keymap/mixin/ControlsOptionsScreenMixin.java [#keymap:keymap-common.mixins.json]:66) ~[client-intermediary.jar:?]
	at net.minecraft.class_4185.method_25306(class_4185.java:40) ~[client-intermediary.jar:?]
	at net.minecraft.class_4264.method_25348(class_4264.java:16) ~[client-intermediary.jar:?]
	at net.minecraft.class_339.method_25402(class_339.java:121) ~[client-intermediary.jar:?]
	at net.minecraft.class_4069.method_25402(class_4069.java:27) ~[client-intermediary.jar:?]
	at net.minecraft.class_312.method_1611(class_312.java:94) ~[client-intermediary.jar:?]
	at net.minecraft.class_437.method_25412(class_437.java:492) ~[client-intermediary.jar:?]
	... 17 more
```

**Comment by Einjerjar** [comment]
Can you provide me with the version of Minecraft this was encountered on, along with version of the mod? thanks!

**Comment by SettingDust** [comment]
keymap-0.7.0-beta.0+1.18.2-fabric

**Comment by SettingDust** [comment]
config file
```json
{
  "firstOpenDone": false,
  "autoSelectLayout": false,
  "customLayout": "en_us",
  "replaceKeybindScreen": true,
  "malilibSupport": true,
  "debug": false,
  "debug2": false,
  "crashOnProblematicError": false,
  "showHelpTooltips": true
}
```

**Comment by Einjerjar** [comment]
Got this, thanks! Ill try working on this later today

**Comment by Einjerjar** [comment]
Should be fixed on the 0.7.0-beta.1 build over at modrinth and cf (_might still be waiting for approval_), thanks!

**Comment by SettingDust** [comment]
> Should be fixed on the 0.7.0-beta.1 build over at modrinth and cf (_might still be waiting for approval_), thanks!

Fixed
